### DISCUSSION TITLE
Rename slingbox 500 from 'My Slingbox' to something else

### DISCUSSION THREAD

**Discussion by egadget1** [GerryDazoo/Slinger#149] [General]
Has anyone done this yet?  I have two connected and there both named 'My Slingbox'

Thanks

**Comment by GerryDazoo** [comment]
Use slingplayer desktop to reconfigure.

**Comment by GerryDazoo** [comment]
Looks like you can't change the name on the 500. Is that what you have?

**Comment by egadget1** [comment]
Hi,
   Yes the slingplayer desktop only gets to the remote setup and I can't pick/search from the devices.

**Comment by sp222222** [comment]
not true _ I have two 500s one I bought last month and both have been renamed using http://newwatch.slingbox.com/
